### ISSUE TITLE
Model can not converge when using other dataset.

### ISSUE THREAD

**Issue by hirabarahyt** [EvelynFan/FaceFormer#111]
Thanks for your great work. I have reproduced the result after following the readme on voca dataset.
However, when I try to do some modification to the structure and use my own dataset, the model seems can not converge and audio data seems not working.
Does anyone meet this kind of problem?

**Comment by hirabarahyt** [comment]
I think I found the problem,
The wav2vec model seems only freeze the feature extractor part:
https://github.com/EvelynFan/FaceFormer/blob/dfaea81983665b22b99af336a80574208cfcc099/faceformer.py#L73
when changing something or using a higher learning rate may break the well trained wav2vec, and then make the audio input not working.

Therefore, when using your own dataset or doing modification at model structure or using high learning rate to train this model from stracth, I recommend you to freeze all the parameters of wav2vec:
`for param in self.audio_encoder.parameters():`
`    param.requires_grad = False`
`self.audio_encoder.eval()`
Then after the whole model is well-trained, only freeze the extractor of wav2vec, using a low learning rate (less than 1e-4 maybe) to finetune both the model and wav2vec.

This should solve #42, and may help to solve #107, #105, #76.

Directly using a lower learning rate for training from scratch may also solve this problem.
Hope someone give some feedbacks.
